docs/src/AlgebraicGeometry/Surfaces/SurfacesP4.md

```diff
@@ -0,0 +1,320 @@
+```@meta
+CurrentModule = Oscar
+DocTestSetup = quote
+  using Oscar
+end
+```
+
+```@setup oscar
+using Oscar
+```
+
+```@contents
+Pages = ["SurfacesP4.md"]
+```
+
+# Nongeneral Type Surfaces in $\mathbb P^4$
+
+## Rational Surfaces
+
+#### A Rational Surface with $d=3$, $\pi=0$
+
+```@docs
+cubic_scroll()
+```
+
+#### A Rational Surface with $d=4$, $\pi=0$
```

Comment:
Wouldn't it make more sense to move this description into the respective docstrings, so that REPL users can also see it?

(I am fine either way, just wondering)

Reply:
But this information is in the respective doc strings. I do not understand?